src/components/mermaidInputs/InputMuiChipSelectWithLabelAndValidation/InputMuiChipSelectWithLabelAndValidation.js

```diff
@@ -63,7 +62,6 @@ const InputMuiChipSelectWithLabelAndValidation = ({
               key={option.value}
               value={option.value}
               sx={{ fontSize: theme.typography.defaultFontSize }}
-              style={getMenuItemStyle({ option, selectedItems: value, theme })}
```

Comment:
removed this because we don't use bolding in the method filter drop down - still need Al to look over it to confirm